tools/pythonpkg/src/include/duckdb_python/pandas_type.hpp

```diff
@@ -67,9 +67,11 @@ enum class PandasType : uint8_t {
 	PANDA_DATETIME = 101,       //! datetime64[ns, UTC]
 	PANDA_PERIOD = 102,         //! datetime64[ns, UTC]
 	PANDA_INTERVAL = 103,       //! datetime64[ns, UTC]
+	PANDA_DATETIME_TZ = 104,    //! datetime64[ns, TZ]
```

Comment:
These are the `num`'s of the dtypes used by Pandas when extending the NumpyDtype.
At some point I planned to use these instead of the conversions to string, to find out which dtype we're dealing with.

But that turned out to not be possible, so I should have removed the explicit values for the dtypes, as they are now unused.

Reply:
@Tishj oh, I meant where in the pandas codebase - I couldn't find any reference to these "major numbers"